lib/api/convex.ts

```
@@ -29,20 +29,37 @@ export const makeAuthenticatedRequest = async (
-    },
+    headers,
   };
 
+  // Handle different types of data
   if (data) {
-    requestOptions.body = JSON.stringify(data);
+    if (data instanceof File || data instanceof Blob) {
+      // For file uploads, don't set Content-Type (let browser set it with boundary for multipart)
+      // But if custom Content-Type is provided, use it
+      if (!customHeaders?.['Content-Type']) {
+        delete headers['Content-Type'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)